build.sh

```diff
@@ -2,19 +2,44 @@
 
 set -o verbose
 
+if [ -n "${DOCKER_IMAGE}" ]; then
+
+    docker pull ${DOCKER_IMAGE}
+    docker run --env CUSTOM_BUILD_SCRIPT --env CUSTOM_TEST_SCRIPT --env SWIFT_SNAPSHOT --env MYSQL_VER -v ${TRAVIS_BUILD_DIR}:${TRAVIS_BUILD_DIR} ${DOCKER_IMAGE} /bin/bash -c "apt-get update && apt-get install -y apt-utils debconf-utils dialog git sudo lsb-release wget libxml2 && cd $TRAVIS_BUILD_DIR && ./build.sh"
 
-if [[ $TRAVIS_OS_NAME == "osx" ]]; then
-    mysql --version || { brew update && brew install mysql@5.7 && brew link mysql@5.7 --force && mysql.server start && mysql --version; }
 else
-    export DEBIAN_FRONTEND="noninteractive"
-    mysql --version || { apt-get update && apt-get install -y mysql-server libmysqlclient-dev && service mysql start && mysql --version; }
-fi
 
-mysql_upgrade -uroot || echo "No need to upgrade"
-mysql -uroot -e "CREATE USER 'swift'@'localhost' IDENTIFIED BY 'kuery';"
-mysql -uroot -e "CREATE DATABASE IF NOT EXISTS test;"
-mysql -uroot -e "GRANT ALL ON test.* TO 'swift'@'localhost';"
+    if [[ $TRAVIS_OS_NAME == "osx" ]]; then
+        if [[ $MYSQL_VER == 5 ]]; then
+            mysql --version || { brew update && brew install mysql@5.7 && brew link mysql@5.7 --force && mysql.server start && mysql --version; }
+        else
+            mysql --version || { brew update && brew install mysql && mysql.server start && mysql --version; }
+        fi
+    else
+        export DEBIAN_FRONTEND="noninteractive"
+        if [[ $MYSQL_VER == 5 ]]; then
+            mysql --version || { apt-get update && apt-get install -y pkg-config mysql-server libmysqlclient-dev && service mysql start && mysql --version; }
+        else
+            cd /tmp
+            wget https://dev.mysql.com/get/mysql-apt-config_0.8.10-1_all.deb
+            cd -
+            sudo -E echo mysql-apt-config mysql-apt-config/select-server select mysql-8.0 | debconf-set-selections
```

Comment:
This doesn't seem to be installing msql v8 - according to the Travis job:
https://travis-ci.org/IBM-Swift/SwiftKueryMySQL/jobs/437637108
> `export MYSQL_VER=8`
> ...
> `mysql  Ver 14.14 Distrib 5.7.23, for Linux (x86_64)`

Perhaps it relates to this failure:
```
Selecting previously unselected package mysql-apt-config.
dpkg: regarding .../mysql-apt-config_0.8.10-1_all.deb containing mysql-apt-config, pre-dependency problem:
 mysql-apt-config pre-depends on gnupg
  gnupg is not installed.
dpkg: error processing archive /tmp/mysql-apt-config_0.8.10-1_all.deb (--install):
 pre-dependency problem - not installing mysql-apt-config
Errors were encountered while processing:
 /tmp/mysql-apt-config_0.8.10-1_all.deb
```

Reply:
Fixed